### PR TITLE
Update output

### PR DESCRIPTION
**Pull request by oldLady344**
updated the output of all the commands through the doc. 

WHY: Three graphd services are now deployed during the deployment of Nebula Graph with Docker Compose.

### CODE REVIEW

**Review by whitewum — COMMENTED** [review]
image
sth wrong for preview.

File: docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md [hunk 1]
```diff
@@ -93,15 +96,50 @@ Run `docker-compose ps` to list the service information of Nebula Graph, such as
...
-nebula-docker-compose_metad1_1      ./bin/nebula-metad --flagf ...   Up (healthy)   0.0.0.0:32776->11000/tcp, 0.0.0.0:32774->11002/tcp, 0.0.0.0:32771->45500/tcp, 45501/tcp
-nebula-docker-compose_metad2_1      ./bin/nebula-metad --flagf ...   Up (healthy)   0.0.0.0:32772->11000/tcp, 0.0.0.0:32769->11002/tcp, 0.0.0.0:32768->45500/tcp, 45501/tcp
-nebula-docker-compose_storaged0_1   ./bin/nebula-storaged --fl ...   Up (healthy)   0.0.0.0:32780->12000/tcp, 0.0.0.0:32778->12002/tcp, 0.0.0.0:32777->44500/tcp, 44501/tcp
-nebula-docker-compose_storaged1_1   ./bin/nebula-storaged --fl ...   Up (healthy)   0.0.0.0:32785->12000/tcp, 0.0.0.0:32784->12002/tcp, 0.0.0.0:32783->44500/tcp, 44501/tcp
-nebula-docker-compose_storaged2_1   ./bin/nebula-storaged --fl ...   Up (healthy)   0.0.0.0:32782->12000/tcp, 0.0.0.0:32781->12002/tcp, 0.0.0.0:32779->44500/tcp, 44501/tcp
+        Name                 Command             State              Ports       
+--------------------------------------------------------------------------------
+nebula-docker-         ./bin/nebula-graphd    Up (healthy)   0.0.0.0:32963->1300
```
Comment (whitewum) [inline docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md:101]: why a new line?
looks wired.

**Review by whitewum — COMMENTED** [review]
2 minors

**Comment by whitewum** [comment]
image
 Even a screenshot is easier to read.

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**